plugins/sdm/lib/helper/deny_helper.py

```diff
@@ -0,0 +1,46 @@
+import shortuuid
+
+
+class DenyHelper:
+    def __init__(self, bot):
+        self.__bot = bot
+        self.__sdm_service = bot.get_sdm_service()
+
+    def execute(self, user, request_id, denial_reason):
+        execution_id = shortuuid.ShortUUID().random(length=6)
+        self.__bot.log.debug("##SDM## %s DenyHelper.execute request_id: %s", execution_id, request_id)
+
+        if not self.__bot.is_valid_grant_request_id(request_id):
```

Comment:
Think about reusing this. Many pieces of this class, look "familiar" from ApproveHelper.